test/manual/api/backup_restore_test.sh

```
@@ -0,0 +1,163 @@
+
+# File to store backup data
+BACKUP_FILE="/tmp/wanderer_map_backup.json"
+
+# ─── UTILITY FUNCTIONS ─────────────────────────────────────────────────────
+
+# Function to backup current map state
+backup_map_state() {
+  echo "==== Backing Up Map State ===="
+  
+  echo "Fetching current map state..."
+  local raw=$(make_request GET "$API_BASE_URL/api/maps/$MAP_SLUG/systems")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)